project/Build.scala

```diff
@@ -1069,6 +1069,49 @@ object Build extends sbt.Build {
   // Testing
 
   val testTagSettings = Seq(
+      javaOptions in Test ++= {
```

Comment:
The common logic between `javaOptions` and `testOptions` should somehow be factored out.